### ISSUE TITLE
Please, update dependencies

### ISSUE THREAD

**Issue by m-valkov** [ivanmarban/winston-telegram#31]
```
# npm audit report

striptags  <3.2.0
Severity: moderate
Passing in a non-string 'html' argument can lead to unsanitized output - https://github.com/advisories/GHSA-qxg5-2qff-p49r
fix available via `npm audit fix --force`
Will install winston-telegram@2.4.0, which is a breaking change
node_modules/striptags
  utils  >=0.3.1
  Depends on vulnerable versions of striptags
  node_modules/utils
    sf  >=0.2.0
    Depends on vulnerable versions of utils
    node_modules/sf
      winston-telegram  1.3.1 - 2.3.5 || >=2.4.1
      Depends on vulnerable versions of sf
      node_modules/winston-telegram
```

**Comment by m-valkov** [comment]
#32

**Comment by ivanmarban** [comment]
Fixed on v2.6.0
